docs/tutorial/a_minimal_working_example.rst

```diff
@@ -0,0 +1,128 @@
+A minimal working example
+=========================
+
+We start with a minimal working example of ALF. The example, as a pure ALF
+configuration file, is located at ``<ALF_ROOT>/alf/examples/tutorial/minimal_example_conf.py``,
+and consists of only 8 lines.
+
+Train and play
+--------------
+
+Let's ignore its content for a moment (see the next section
+:doc:`./understanding_ALF_via_the_minimal_working_example` for an explanation of
+the configuration content), and just focus on how to launch the training,
+interpret the output training messages, and play a trained model.
+
+Train from scratch
+^^^^^^^^^^^^^^^^^^
+
+We can train from scratch by
```

Comment:
Do you think it might benefit the first-time users if the concept of `root dir` (a.k.a. state directory, a.k.a. operating direcotry)?